python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -7447,73 +7455,52 @@ def test_dataframe_sample_basic(n, frac, replace, axis):
                     "n": n,
                     "frac": frac,
                     "replace": replace,
-                    "random_state": random_state,
+                    "random_state": gd_random_state,
                     "axis": axis,
                 },
             ),
         )
     else:
-        gout = df.sample(
+        got = df.sample(
             n=n,
             frac=frac,
             replace=replace,
-            random_state=random_state,
+            random_state=gd_random_state,
             axis=axis,
         )
-        assert pout.shape == gout.shape
+        checker(expected, got)
 
 
 @pytest.mark.parametrize("replace", [True, False])
-@pytest.mark.parametrize("random_state", [1, np.random.mtrand.RandomState(10)])
-def test_dataframe_reproducibility(replace, random_state):
+def test_dataframe_reproducibility_seed(replace):
```

Comment:
Is this test redundant? `test_dataframe_sample_reproducibility_seed` seems to cover the same things, except a bit better?

Reply:
yup